packages/front-end/package-lock.json

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
We use `yarn` instead of `npm` to manage dependencies.  Delete this file and run `yarn install` instead.  It will update the `yarn.lock` file with your new dependencies.